docs/self-managed/zeebe-deployment/exporters/camunda-exporter.md

```diff
@@ -0,0 +1,189 @@
+:::
+
+</TabItem>
+
+<TabItem value="security">
+
+| Option          | Description                                                                                       | Default |
+| --------------- | ------------------------------------------------------------------------------------------------- | ------- |
+| enabled         | If `true`, enables the security (ssl) features for the exporter.                                  | false   |
+| certificatePath | The file path to the SSL certificate used for secure communication with Elasticsearch/Opensearch. |         |
+| verifyHostname  | If `true`, the hostname of the SSL certificate will be validated.                                 | true    |
+| selfSigned      | If `true`, allows the use of self-signed SSL certificates.                                        | false   |
+
+</TabItem>
+
+<TabItem value="index">
+
+| Option                | Description                                                                                                                                                                                 | Default |
+| --------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ------- |
+| prefix                | This prefix will be appended to every index created by the exporter; must not contain `_` (underscore).                                                                                     |         |
+| numberOfShards        | The number of [shards](https://www.elastic.co/guide/en/elasticsearch/reference/current/index-modules.html#_static_index_settings) used for each new record index created.                   | 3       |
+| numberOfReplicas      | The number of shard [replicas](https://www.elastic.co/guide/en/elasticsearch/reference/current/index-modules.html#dynamic-index-settings) used for each new record index created.           | 0       |
+| variableSizeThreshold | Defines a threshold for variable size. Variables exceeding this threshold are split into two properties: `FULL_VALUE` (full content, not indexed) and `VALUE` (truncated content, indexed). | 8191    |
+| shardsByIndexName     | A map where the key is the index name and the value is the number of shards, allowing you to override the default `numberOfShards` setting for specific indices.                            |         |
+| replicasByIndexName   | A map where the key is the index name and the value is the number of replicas, allowing you to override the default `numberOfReplicas` setting for specific indices.                        |         |
```

Comment:
❓ This means setting this would also win right?

Reply:
yes, the ones in the map will override the default values set above